.circleci/config.yml

```diff
@@ -1,71 +1,26 @@
 version: 2.1
+
 commands:
   requirements:
-    parameters:
-      python_version:
-        type: "string"
-        default: "3.5"
     steps:
-      - run:
-          name: Install virtualenv
-          command: sudo apt-get update && sudo apt-get -y -q install virtualenv
-      - run:
-          name: Create a virtualenv environment
-          command: "virtualenv -p python<<parameters.python_version>> venv"
-
       - run:
           name: Install requirements
-          command: ". venv/bin/activate && make requirements"
+          command: "sudo make requirements"
```

Comment:
@giovannicimolin, personally, I  like using virtualenvs in the CI, as it allows easier caching [like this](https://gitlab.com/opencraft/dev/sprints/-/blob/master/.gitlab-ci.yml#L44-53).

Reply:
@Agrendalath Makes sense, I've reverted back to a virtualenv and added the `venv` folder to the caching settings.